.test_config

```diff
@@ -0,0 +1,5 @@
+{
+  "test_package": {
+    "platforms": ["vm"]
+  }
+}
```

Comment:
??? Do we even care about this file any more?

Reply:
nope, must have missed this in one of the repos.